config/v1/types_infrastructure.go

```diff
@@ -118,6 +122,15 @@ type AWSPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// BareMetalPlatformStatus holds the current status of the BareMetal infrastructure provider.
+type BareMetalPlatformStatus struct {
+	// api_vip holds the VIP that can be used to contact the API
+	APIVIP string `json:"api_vip"`
```

Comment:
What's VIP?

Reply:
> What's VIP?

Virtual IP

Reply:
Can you expand that in the comment in that case?

Reply:
Will do.